### PR TITLE
60424 - Hide analytic settings panel when manageAnalyticAccounting is…

### PR DESCRIPTION
**Pull request by paa-axelor**
… not checked on company settings

### CODE REVIEW

File: axelor-account/src/main/resources/views/Account.xml [hunk 1]
```diff
@@ -120,8 +120,9 @@
           <field name="vatSystemSelect" colSpan="4"
             showIf="isTaxAuthorizedOnMoveLine &amp;&amp; (accountType.technicalTypeSelect == 'immobilisation' || accountType.technicalTypeSelect == 'charge' || accountType.technicalTypeSelect == 'income')"/>
         </panel>
-        <panel name="analyticSettingsPanel" title="Analytic" colSpan="12"
-          hideIf="!accountType || accountType.technicalTypeSelect == 'view'">
+        <panel
+          if="__config__.app.getApp('account').getManageAnalyticAccounting() || (!accountType &amp;&amp; accountType.technicalTypeSelect != 'view')"
```
Comment (jmi-axelor) [inline axelor-account/src/main/resources/views/Account.xml:124]: I think it's : 
`if="__config__.app.getApp('account').getManageAnalyticAccounting() &amp;&amp; accountType.technicalTypeSelect != 'view'"`

**Review by jmi-axelor — CHANGES_REQUESTED** [review]

File: axelor-account/src/main/resources/views/Account.xml (hunk 1 above)
Comment (paa-axelor) [reply axelor-account/src/main/resources/views/Account.xml]: Solved

**Review by jmi-axelor — APPROVED** [review]

**Review by jmi-axelor — APPROVED** [review]

File: axelor-account/src/main/java/com/axelor/apps/account/web/AccountController.java [hunk 2]
```diff
@@ -112,11 +112,15 @@ public void checkAnalyticAccount(ActionRequest request, ActionResponse response)
...
+      response.setAttr("analyticSettingsPanel", "hidden", false);
+
       if (account.getCompany() == null
           || !Beans.get(AppAccountService.class).getAppAccount().getManageAnalyticAccounting()
           || !Beans.get(AccountConfigService.class)
               .getAccountConfig(account.getCompany())
-              .getManageAnalyticAccounting()) {
+              .getManageAnalyticAccounting()
+          || account.getAccountType() == null
+          || "view".equals(account.getAccountType().getTechnicalTypeSelect())) {
```
Comment (ale-axelor) [inline axelor-account/src/main/java/com/axelor/apps/account/web/AccountController.java:123]: please use static variable AccountTypeRepository.TYPE_VIEW here

**Review by ale-axelor — CHANGES_REQUESTED** [review]

File: axelor-account/src/main/java/com/axelor/apps/account/web/AccountController.java (hunk 2 above)
Comment (paa-axelor) [reply axelor-account/src/main/java/com/axelor/apps/account/web/AccountController.java]: Resolved

**Pull request merged**